app/controllers/deploys_controller.rb

```diff
@@ -54,7 +63,7 @@ def destroy
     unless flash.now[:error]
       flash.now[:notice] = 'Terraform resources have been destroyed.'
     end
-    render :show
+    redirect_to action: 'show'
```

Comment:
why do we change this?

Reply:
Because `render` only just render the show view of the controller, without running any other operation.
Now, the `show` controller executes some operations (get the provisioners data), so if we don't run these actions, some data is not populated and the show fails.
Summary, after changing the `show` controller, rendering is not enough, it must be redirected